_data/versions.yml

```diff
@@ -1346,6 +1346,102 @@ v2.1:
       version: 1.4.1
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.1
 
+v3.0:
+- title: v3.0.0-beta1
+  note: |
+    17 November 2017
+    
+    This is a pre-release of v3.0.0. This release is intended for testing purposes only and is NOT to be used on production systems.
+
+    #### What's new
+
+    - Support for the [etcd version 3 datastore](https://coreos.com/blog/etcd3-a-new-etcd.html).
```

Comment:
I think we should say `Support for the etcd v3 API` or `Support for etcdv3`.

I think "etcd version 3 datastore" is a bit atypical.

Reply:
OK yes, now that I'm looking into this a bit more, it seems that you're right, thanks!